lib/src/base-styles/_nx-card.scss

```diff
@@ -14,91 +14,62 @@
   border-radius: $nx-border-radius;
   box-sizing: border-box;
   margin-bottom: $nx-spacing-md;
-  padding: $nx-spacing-l $nx-spacing-xxl;
-}
-
-.nx-card__call-out {
-  @include bold();
-
-  font-size: 32px;
-
-  .nx-icon--card-call-out {
-    font-size: 48px;
-  }
-
-  .nx-icon--card-call-out-xl {
-    font-size: 64px;
-  }
+  padding: $nx-spacing-l;
+  text-align: center;
 }
 
 .nx-card-container--row {
+  @include container-horizontal;
   display: flex;
-  -ms-flex-wrap: wrap;
   flex-wrap: wrap;
+  justify-content: space-between;
   margin-bottom: 0;
+}
 
-  + .nx-card-container--row {
-    margin-top: 0;
-  }
-
-  .nx-card {
-    @include container-vertical;
-    display: flex;
-    flex-direction: column;
-    -ms-flex-pack: justify;
-    justify-content: space-between;
-    margin-right: $nx-spacing-md;
-    text-align: center;
-    min-width: 230px;
-    max-width: 330px;
-  }
-
-  .nx-card__header {
-    @include container-vertical;
-    margin-bottom: $nx-spacing-md;
-  }
+.nx-card {
+  @include container-vertical;
+  display: grid;
+  grid-template-rows: auto 1fr auto;
+  margin-right: $nx-spacing-md;
+  min-width: 230px;
+  max-width: 330px;
```

Comment:
We'll probably want to make it clear in the documentation that this is a good default but which the consumer would often override for deliberately-wide cards.  Or maybe we should even make a `--wide` variant?

Reply:
> We'll probably want to make it clear in the documentation that this is a good default but which the consumer would often override for deliberately-wide cards. Or maybe we should even make a `--wide` variant?

We'll never get a `--wide` variant right. Some design will always need it narrower or wider than whatever magic number we choose. I'd rather just leave it custom and demonstrate how to set that custom value.

Reply:
> We'll never get a --wide variant right. Some design will always need it narrower or wider than whatever magic number we choose. I'd rather just leave it custom and demonstrate how to set that custom value.

I see we've now got an example of a card that expands as much as possible which is great, but do you think we also need an example of a card with a specific custom max-width, or do you think consumers will be able to figure that out on their own?